Server.js

```diff
@@ -1,10 +1,14 @@
 const express = require('express')
 const fs = require('fs')
+const db = require('./db.js')
 
 const app = express()
-const port = 80
+app.use(express.json())
 
 var new_task_id = 0
+const port = 80
+
+require('./Routs/tasksRouts.js')(app, db)
```

Comment:
you don't need to require here the db and then forwored it to the tasksRoute, you can just require it in the tasksRoute